### ISSUE TITLE
name of the loadbalancer machine in the master-1's hosts file is lb, but we use the name loadbalancer while connecting to it from vagrant

### ISSUE THREAD

**Issue by viveksinghggits** [mmumshad/kubernetes-the-hard-way#31]
So, the command that we use to connect to the loadbalancer machine using vagrant is 
```vagrant ssh loadbalancer```
but when we try to connect to this machine through ssh from machine `master-1` 
```
vagrant@master-1:~$ ssh loadbalancer
```
it gives us below error
```
ssh: Could not resolve hostname loadbalancer: Temporary failure in name resolution
```
and the reason is that in `/etc/hosts` file the name of the host is `lb`, i think we should get that changed to `loadbalancer` to avoid confusion.

**Comment by fgauna12** [comment]
good catch, I got confused by this

**Comment by fireflycons** [comment]
Hi @viveksinghggits @fgauna12 

We have now merged a major change to bring this to v1.24.
Many issues have been addressed as part of the upgrade. 
I realise this is a very old issue, however I was tasked with closing off all old issues following the upgrade.
Please try it now and feel free to raise further issues.

Thanks.
